### PR TITLE
Add preregistration

### PR DESCRIPTION
**Pull request by chartgerink**
This addresses #15 

# Reviewer notes

* Limited information on all the material behind preregistration, but this is supposed to be an onboarding after all 👍 Let me know if something crucial is missing?

# New topics

These are topics not previously mentioned in the content or issues and that need to be added in the issues. Please use checkmarks so maintainers can keep track of which need to be added.

- [x] Preservation

# External reviewers

Feel free to review if you want, feel free to not do so too. Much love to reviewers for their willingness to take part 😍 

@jibe-b
@amscheel
@emaassen

### CODE REVIEW

**Comment by jibe-b** [comment]
I unfortunately don't know enough about the pros and cons of preregistration

Here are two possible readings. TL;DR: preregistration is a good step but not yet enough.

www.the100.ci/2017/09/14/overfitting-vs-open-data/

http://effective-altruism.com/ea/121
/a_method_for_automatic_trustworthiness_in_study/

File: content/preregistration.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Preregistration
+
+Predictions are tested on data; preregistration helps prove that predictions precede the data. This chronology is essential for valid tests of predictions (confirmatory research). Preregistration is not necessary for research that has no predictions and uses no hypothesis testing (exploratory research). Preregistration derives its name from the act of registering predictions before (pre) a publication finds place (not to be confused with [Preprints](preprints.md)).
```
Comment (rainsworth) [inline content/preregistration.md]: "publication finds place..." -> A little confusing wording. 

Should preregistration happen at the start of a project, before the data collection/analysis and long before publication takes shape? I wonder if this varies across fields, but for astronomy I would assume preregistration would take place around the proposal stage.

Perhaps (although feel free to ignore): "registering predictions before (pre) analysis" ?

File: content/preregistration.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+To show the necessity of preregistration, consider this analogy: Horse racing bets cannot be placed after the results are known. Predictions in research are bets on how well a phenomenon is understood; the more specific the prediction the higher the gains if correct. However, it also means that bets cannot change, even in the slightest sense.
```
Comment (rainsworth) [inline content/preregistration.md]: 👍  great analogy

**Review by rainsworth — APPROVED** [review]

File: content/preregistration.md (hunk 1 above)
Comment (chartgerink) [reply content/preregistration.md]: Thanks, good suggestion! It does vary probably and your revision gets right to the essence.

**Comment by chartgerink** [comment]
@jibe-b yea there are connotations about preregistration too. Would need a book to be precise about these things.

File: content/preregistration.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+By not preregistering predictions, there is no guarantee that what the prediction is said to be actually was the original prediction. This includes guarantees to the outside world but also to yourself (it is very easy to fool yourself). [A survey](https://doi.org/10.1177/0956797611430953) indicates that certain behaviors that could undermine researcher's own predictions seem to be the norm. 
+
+By preregistering predictions (hypotheses), it is possible to make research more rigorous. However, the benefits of preregistration depend on how thoroughly it is conducted. Moreover, a preregistration is only as worthwhile as the follow through during analysis and reporting of results. As such, doing a preregistration requires the researcher to be as specific as possible and to also include statements that remove all confusion. A good rule of thumb: Preregistrations need to be as specific as the analyses that will be conducted on top of them later.
+<!-- Might consider adding preregistration tips at a later stage -->
```
Comment (emaassen) [inline content/preregistration.md:10]: I would either add a paragraph including preregistration on the different aspects of research (eg: preregistering hypotheses, methods, analysis plan), or add these in the preregistration tips.

File: content/preregistration.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+There are several places on the Web to preregister hypotheses, including the [Open Science Framework](https://osf.io), [Zenodo](https://zenodo.org), or any other service that archives files reliably and with clear timestamps. Such services are currently always third-parties and need to be trusted to provide reliable information and preserve the information into the future (see also [Reference Rot](reference-rot.md) and [Preservation](preservation.md)). Currrently the [Preregistration Challenge](https://cos.io/prereg/) rewards individuals if they preregister a study with $1,000.
```
Comment (emaassen) [inline content/preregistration.md:12]: Curr__r_ently the [Preregistration Challenge](https://cos.io/prereg/) rewards individuals if they preregister a study with $1,000.

I would nuance- Currently individuals preregistering a study _can_ be rewarded with $1,000 by the PreReg Challenge.

File: content/preregistration.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Exercises
+
+* Think about your own research: Do you gather data to test predictions that could be pregistered?
```
Comment (emaassen) [inline content/preregistration.md:16]: Maybe also _what_ can be preregistered (more than predictions)? See also comment above.

**Review by emaassen — COMMENTED** [review]
Expand beyond preregistering hypotheses only. :octocat:

File: content/preregistration.md (hunk 4 above)
Comment (chartgerink) [reply content/preregistration.md:12]: Adopted!

File: content/preregistration.md (hunk 5 above)
Comment (chartgerink) [reply content/preregistration.md:16]: Added.

File: content/preregistration.md (hunk 3 above)
Comment (chartgerink) [reply content/preregistration.md:10]: Added a paragraph.

**Pull request merged**